stripecardscan/res/values/strings.xml

```diff
@@ -9,7 +9,7 @@
     <string name="stripe_card_scan_title" description="Text displayed above the scan window for paymentsheet, instructing the user how to scan card">Position your card in the rectangle to scan</string>
     <string name="stripe_card_scan_instructions" description="Text displayed above the scan window, instructing the user how to scan card">Scan Your Card</string>
     <string name="stripe_card_scan_security" description="Security notification displayed below the scan window">Your card info is secure</string>
-    <string name="stripe_card_scan_privacy_link_text" description="Privacy link text in scan window"><![CDATA[We use Stripe to verify your card details. Stripe may use and store your data according its privacy policy. <a href="https://support.stripe.com/questions/stripes-card-image-verification"><u>Learn more</u></a>]]></string>
+    <string name="stripe_card_scan_privacy_link_text" description="Privacy link text in scan window. THIS STRING SHOULD NOT BE MODIFIED"><![CDATA[We use Stripe to verify your card details. Stripe may use and store your data according its privacy policy. <a href="https://support.stripe.com/questions/stripes-card-image-verification"><u>Learn more</u></a>]]></string>
```

Comment:
does the anchor tag actually work??

Reply:
It does!

Reply:
Not needed in this PR, but could you fix the package name of [TextUtils.kt](https://github.com/stripe/stripe-android/pull/5021/files#diff-1a4422b4d2224a186979525c9b3b6aa9d01dc71c190c1479102bdf1613344b00) and move it to `stripe-core`(probably `com.stripe.android.core.util`) then share it with both `Identity` and `CardScan?